examples/hashtag-example/src/Hashtag.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+
+export default class Hashtag extends Component {
+  render() {
+    const {
+      decoratedText, // eslint-disable-line no-unused-vars
+      dir, // eslint-disable-line no-unused-vars
+      entityKey, // eslint-disable-line no-unused-vars
+      offsetKey, // eslint-disable-line no-unused-vars
+      contentState, // eslint-disable-line no-unused-vars
+      ...otherProps
+    } = this.props // eslint-disable-line no-use-before-define
```

Comment:
no need to do this, replace `{...otherProps}` below with `{...this.props}` that'll do